src/Orleans/Runtime/GrainReference.cs

```diff
@@ -461,24 +461,10 @@ protected GrainReference(SerializationInfo info, StreamingContext context)
                 genericArg = null;
             genericArguments = genericArg;
 
-#if !NETSTANDARD_TODO
-            this.OnDeserialized(context);
-#endif
-        }
-
-#if !NETSTANDARD_TODO
-        /// <summary>
-        /// Called by BinaryFormatter after deserialization has completed.
-        /// </summary>
-        /// <param name="context">The context.</param>
-        [OnDeserialized]
```

Comment:
are you sure getting rid of the method with `[OnDeserialized]` is correct? @ReubenBond can you advise?
BTW, there are more occurrences of this in other files

Reply:
We're using it at 2 places and this change was coordinated with @ReubenBond.

Reply:
I was just being 'extra careful' with the OnDeserialized attribute. The ctor will be called anyway. We can remove it, we have tests which ensure that BinaryFormatter works with GrainReferences.

Reply:
`Serialize_GrainReference_ViaStandardSerializer` is the test